ACE/ace/config-android.h

```diff
@@ -357,6 +357,10 @@
 # define ACE_LACKS_CONDATTR 1
 #endif
 
+#if defined(__BIONIC_CAST)
```

Comment:
Any version number checked required?

Reply:
There's a reasonable argument to be made (not sure if it's 100% correct), that POSIX only guarantees FD_ISSET can work on a pointer to non-const fd_set.  Would it be easier just to make it a mutable member of ACE_Handle_Set?

Reply:
Yeah, that's a reasonable point. But I'd want to avoid making it mutable. That should be a last resort. Alternatively, we could make a local variable copy of the fd_set first, and call FD_ISSET on that. Reasonable compilers should be able to optimize the copy away, at least in release mode.

Reply:
Maybe we should go with the const_cast always, that would make this code a little bit easier to maintain.

Reply:
Yeah, the only risk of undefined behavior would be if someone
1. creates a `const ACE_Handle_Set`, which I wouldn't expect to be common usage, *and*
2. calls an `FD_ISSET` implementation with it which actually modifies the `fd_set` given, which doesn't make sense, and as far as I know, doesn't exist anywhere.

So I'd think always `const_cast` is fine. But if you want absolute conformance with POSIX, I think the local variable copy is the best solution.